### PR TITLE
Initial commit for Radium blockchain node on Ubuntu VM

### PR DESCRIPTION
**Pull request by JJ12880**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@JJ12880__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@JJ12880__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

**Comment by JJ12880** [comment]
Fixed missing parameter values.

**Comment by azurecla** [comment]
Hi __@JJ12880__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by JJ12880** [comment]
Fixed invalid domain name label.

**Comment by azurecla** [comment]
Hi __@JJ12880__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
Hi __@JJ12880__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by JJ12880** [comment]
fixed file in the root

**Comment by azurecla** [comment]
Hi __@JJ12880__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
Hi __@JJ12880__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: radium-blockchain-ubuntu/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+			"type": "string",
+			"metadata": {
+				"description": "Hostname assigned to the Public IP, also used as the VM Name. Must be lowercase. It should match with the following regular expression: ^[a-z][a-z0-9-]{1,61}[a-z0-9]$ or it will raise an error."
+			}
+		},
+		"installMethod": {
+			"type": "string",
+			"defaultValue": "From_Source",
+			"allowedValues": [
+				"From_Source"
```
Comment (singhkay) [inline radium-blockchain-ubuntu/azuredeploy.json]: Is there way to install this from a binary? Building from source can take a lot of time and has the user paying for compute time

File: radium-blockchain-ubuntu/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+			"metadata": {
+				"description": "Allowed IP for RPC connection."
+			}
+		}
+	},
+  "variables": {
+    "imagePublisher": "Canonical",
+    "imageOffer": "UbuntuServer",
+    "OSDiskName": "RadiumOSDisk",
+    "imageSKU": "15.10",
```
Comment (singhkay) [inline radium-blockchain-ubuntu/azuredeploy.json]: Can this work on 14.04 LTS? 15.10 will be EOL soon and will be removed from Azure soon after causing this template to break

File: radium-blockchain-ubuntu/azuredeploy.json (hunk 1 above)
Comment (JJ12880) [inline radium-blockchain-ubuntu/azuredeploy.json]: Yes, for sure. Working on adding that now.

File: radium-blockchain-ubuntu/azuredeploy.json (hunk 2 above)
Comment (JJ12880) [inline radium-blockchain-ubuntu/azuredeploy.json]: Will do.

**Comment by azurecla** [comment]
Hi __@JJ12880__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
Hi __@JJ12880__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

**Comment by JJ12880** [comment]
All requested changes have been made. Not sure why travis-ci build has been failing though.

File: radium-blockchain-ubuntu/azuredeploy.json (hunk 2 above)
Comment (JJ12880) [inline radium-blockchain-ubuntu/azuredeploy.json]: Done

File: radium-blockchain-ubuntu/azuredeploy.json (hunk 1 above)
Comment (JJ12880) [inline radium-blockchain-ubuntu/azuredeploy.json]: done

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**